### PR TITLE
Add `isChromeOS` os-util

### PR DESCRIPTION
**Pull request by sereezha**
issue: https://github.com/yola/ws-editor/issues/4297
env: https://editor.hto-ya.env.yola.net/

## How to test
1. Go to [editor](https://editor.hto-ya.env.yola.net/)
### Expected result
- if OS is ChromeOS:
   - in the header you should see `!!!!!CHROME OS!!!!!` label (just for test purposes)
- if OS is NOT ChromeOS:
   - in the header you should see `!!!!!OTHER OS!!!!!` label (just for test purposes)

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by chobitV** [comment]
Let's update `README.md`

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "yola-bowser",
-  "version": "1.1.1",
+  "version": "1.2.0-rc.sereezha.2",
```
Comment (chobitV) [inline package.json:3]: ⚠️

**Review by chobitV — APPROVED** [review]

**Review by juliamokh — APPROVED** [review]

**Pull request merged**